shell/key-bindings.fish

```diff
@@ -44,7 +44,7 @@ function fzf_key_bindings
     set -q FZF_TMUX_HEIGHT; or set FZF_TMUX_HEIGHT 40%
     begin
       set -lx FZF_DEFAULT_OPTS "--height $FZF_TMUX_HEIGHT $FZF_DEFAULT_OPTS --tiebreak=index --bind=ctrl-r:toggle-sort $FZF_CTRL_R_OPTS +m"
-      history | eval (__fzfcmd) -q '(commandline)' | read -l result
+      history -z | eval (__fzfcmd) --read0 -q '(commandline)' | perl -pe 'chomp if eof' | read -lz result
```

Comment:
Thanks, the patch looks good to me. Just note that fzf currently prints a new line character as `?`, this can be improved in the future.

Yeah, it would be nice if we don't have to use perl here, but if there's no other solution, I'm okay with it. (FWIW, `head -c -1` doesn't work on macOS)